api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto

```diff
@@ -56,4 +57,10 @@ message LeastRequest {
   // Configuration for slow start mode.
   // If this configuration is not set, slow start will not be not enabled.
   config.cluster.v3.Cluster.SlowStartConfig slow_start_config = 3;
```

Comment:
I guess this should also switch to `common.v3.SlowStartConfig`.  Technically, that's a breaking change, but I suspect no one is actually using this yet, so that's probably okay.

@temawi, please confirm that this change would not break anything for grpc-java.